example/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,4 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.applovin.enterprise.apps.demoapp">
```

Comment:
How can we build without package name in Android Manifest?
<img width="1840" alt="image" src="https://user-images.githubusercontent.com/5104410/218956470-9b02c8cc-8ca2-432d-b73f-11409be6595d.png">